electrode-ota-server-dao-mariadb/src/queries/PackageQueries.ts

```diff
@@ -103,4 +103,14 @@ export const PackageQueries = {
                     WHERE id = ?`,
 
     updatePackageTime : `UPDATE package SET update_time = CURRENT_TIMESTAMP(3) WHERE id = ?`,
+
+    getMostRecentPackageIdByDeployment : `SELECT dph.package_id, p.create_time, p.app_version
```

Comment:
how is this different from `getMostRecentPackageIdByDeploymentNoTags`?

Reply:
it's mistake, using the existing query instead.